utils/oath-toolkit/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oath-toolkit
+PKG_VERSION:=2.6.2
+PKG_RELEASE:=1
+PKG_SOURCE:=oath-toolkit-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/oath-toolkit/
```

Comment:
You should use the fault-tolerant round-robin download alias for Savannah.
The URL should be ``` @SAVANNAH/oath-toolkit ```

Example e.g. in https://github.com/openwrt/packages/blob/master/libs/confuse/Makefile